### DISCUSSION TITLE
你好你好

### DISCUSSION THREAD

**Discussion by ssbohet** [share121/inter-knot#4697] [其他]
你好我好大家好呀！

**Comment by ssbohet** [comment]
怎么换头像啊？

**Reply by sensei-is-playing-GenShin** [reply]
这个委托我接了
楼主好，首先点开绳网的右上角博客图标
![微信截图_20240909223435](https://github.com/user-attachments/assets/f7f16879-0c4a-41d3-bbe4-cef2a484aabd)
然后在弹出的新窗口点是现在自己头像的图标
![微信截图_20240909223529](https://github.com/user-attachments/assets/9ece344e-1f01-49eb-bd49-c1785fe0fd1e)
接着再选那个“your porfile”选项
![微信截图_20240909223609](https://github.com/user-attachments/assets/dd5fabb2-59b3-42cc-9c7d-260f8fbb405f)
进去后再点左边的自己头像
![微信截图_20240909223709](https://github.com/user-attachments/assets/96eb5c4a-a0eb-46f5-b427-7a26cbb15179)
再进去后点那个右侧自己头像那里的“edit”
![微信截图_20240909223818](https://github.com/user-attachments/assets/52bc99cb-e21a-4cd7-959f-20c336bc71fb)
最后再点一下那个“upload a photo”就可以了
![微信截图_20240909223848](https://github.com/user-attachments/assets/f74cdd11-fe61-4e0b-8bbb-364930197d50)

**Comment by ID104772508** [comment]
太强了，感谢加崇拜！
